incubator/hnc/pkg/kubectl/config.go

```diff
@@ -0,0 +1,34 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Manipulates the HNC configuration",
+	Args:  cobra.MinimumNArgs(1),
```

Comment:
Why do we need this line? What happens if we leave it out? Do we get a description of all the valid subcommands (that's what we want, I think, ideally).

Reply:
Good catch! I think I mixed up "argument" and "subcommand". We can leave this line out. It will show valid subcommands with or without this line, like below:

```
Manipulates the HNC configuration

Usage:
  kubectl config [command]

Available Commands:
  describe    Displays information about the HNC configuration
```